#!/usr/bin/env python
PACKAGE = "waypoint_patroller"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("low_battery", int_t, 0, "low battery treshold", 35, 25, 95)
gen.add("very_low_battery", int_t, 0, "panic treshold. robot should callfor help if battery drops below this level", 20, 10, 30)
gen.add("charged_battery", int_t, 0, "undocking treshold", 98, 40, 100)
gen.add("max_bumper_recovery_attempts", int_t, 0, "numer of bumper recovery attempts before calling for help by sending an email.", 10, 5, 20)
gen.add("max_move_base_recovery_attempts", int_t, 0, "numer of navigation recovery attempts before giving up and tryingto go to another waypoint.", 5, 1, 20)




exit(gen.generate(PACKAGE, "waypoint_patroller", "PatrollerTresholds"))